### ISSUE TITLE
ConversationChain Validation Error

### ISSUE THREAD

**Issue by hussamsayeed** [langchain-ai/langchain#2024]
II'm using some variable data in my prompt, and sending that prompt to ConversationChain and there I'm getting some validation error.
what changes we can make it to work as expected?
![Screenshot (1225)](https://user-images.githubusercontent.com/46478199/227850617-e22658f1-5ebf-41d4-a672-209dd92f5674.png)

**Comment by L4rryFisherman** [comment]
In your memory function, you can define:

- `memory_key` (the identifier for the memory) 
- `input_key` (the identifier for the 'human')
- `output_key` (the identifier for the A.I.

It looks like your prompt's `input_key`: `input` does not match your memory's `input_key`: `history`. 

You can either adjust your memory, like so:

```
memory = ConversationTokenBufferMemory(
    ...
    memory_key="chat_history_lines",
    input_key="input",
    ...
)
```

**Comment by hussamsayeed** [comment]
> In your memory function, you can define:
> 
> * `memory_key` (the identifier for the memory)
> * `input_key` (the identifier for the 'human')
> * `output_key` (the identifier for the A.I.
> 
> It looks like your prompt's `input_key`: `input` does not match your memory's `input_key`: `history`.
> 
> You can either adjust your memory, like so:
> 
> ```
> memory = ConversationTokenBufferMemory(
>     ...
>     memory_key="chat_history_lines",
>     input_key="input",
>     ...
> )
> ```

yep, it is defined exactly the sameway. still it persists the same

**Comment by edom18** [comment]
I have same issue. I've digged the class to know about. I also use ChatPromptTemplate to create prompts with ChatOpenAI class.

I've noticed that the ConversationChain validator will collect all input keys from prompts but the validator refers only its input key and memory key.

Other words, how to add some keys that are hold on prompts to the ConversationChain class?

If the class accepts input keys as an array, I can adjust the keys to that I want.
